### PR TITLE
Adding neptune-client integrations with ML frameworks (multiple packages)

### PR DESCRIPTION
**Pull request by PiotrJander**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by HubertJaworski** [comment]
I confirm

**Comment by aniezurawski** [comment]
I confirm that I am willing to be listed as a maintainer of the submitted packages

File: recipes/neptune-xgboost/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  noarch: python
+  number: 0
+  script: '{{ PYTHON }} -m pip install . -vv '
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - matplotlib
```
Comment (dopplershift) [inline recipes/neptune-xgboost/meta.yaml:22]: Unless this needs/wants the `pyqt` GUI, better to depend on `matplotlib-base`.

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just a couple minor things. @pitercl @shnela @pkasprzyk can you all confirm you are willing to be maintainers on these recipes?

**Comment by pitercl** [comment]
> Just a couple minor things. @pitercl @shnela @pkasprzyk can you all confirm you are willing to be maintainers on these recipes?
I do confirm. Thanks!

**Comment by pkasprzyk** [comment]
I also confirm I am willing to be listed as a maintainer of the submitted packages

**Comment by shnela** [comment]
> Just a couple minor things. @pitercl @shnela @pkasprzyk can you all confirm you are willing to be maintainers on these recipes?

I do.

**Comment by PiotrJander** [comment]
@dopplershift we now have comments with consent from all team members, and I addressed your other comments, but Windows build fails due to seemingly unsatisfiable dependencies. Could you have a look and help us find out how to fix the problem? I'm asking because it is not clear to me in what way exactly version constraints are incompatible, and why this problem only occurs on Windows and not on the other platforms.

**Comment by dopplershift** [comment]
The obvious failure would be because tensorflow 2 isn't available from conda-forge on windows. I'm happy to merge these since they're all noarch python. Once they're built in their own feedstocks, it will be easier for you all to test and see if there are any unexpected dependency failures installing on Windows.

**Pull request merged**